core/sql/exp/ExpLOBaccess.cpp

```diff
@@ -2285,7 +2338,7 @@ Ex_Lob_Error ExLobsOper (
 { 
   Ex_Lob_Error err = LOB_OPER_OK;
   ExLob *lobPtr = NULL;
-  char fn[MAX_LOB_FILE_NAME_LEN];
+  string fn;
```

Comment:
Missed this one. This needs to be converted to a dynamic char array or an NAString on the Lob globals heap.